web-server/v0.4/src/services/explore.js

```diff
@@ -1,6 +1,7 @@
 import request from '../utils/request';
 
-export default async function querySharedSessions(params) {
+// queies all the available shared sessions from the database to display
```

Comment:
We have a spelling error on this comment.

Reply:
Resolved!